packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -67,7 +67,7 @@ export default class ChildProcessWorker implements WorkerInterface {
       cwd: process.cwd(),
       env: {
         ...process.env,
-        JEST_WORKER_ID: String(this._options.workerId),
+        JEST_WORKER_ID: String(this._options.workerId + 1),
```

Comment:
maybe add a comment like `// JEST_WORKER_ID is 1-indexed, while workerId is 0-indexed`?